### PR TITLE
V3/modal stories

### PR DESCRIPTION
**Pull request by abea**
This should be the last of the stories from the storybook repo.

### CODE REVIEW

**Comment by abea** [comment]
I kept it under 40 files, so... you're welcome?

File: lib/modules/@apostrophecms/doc-type/src/apos/components/AposDocEditor.vue [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+            @input="update" :modifiers="['small', 'inverted']"
+          />
+        </div>
+      </AposModalRail>
+    </template>
+  </AposModal>
+</template>
+
+<script>
+import AposModalParentMixin from '../../../../modals/src/apos/mixins/AposModalParentMixin';
```
Comment (abea) [inline lib/modules/@apostrophecms/doc-type/src/apos/components/AposDocEditor.vue:50]: The `'Modules/@apostrophecms'` alias wasn't working for me for script imports.

Comment (boutell) [reply lib/modules/@apostrophecms/doc-type/src/apos/components/AposDocEditor.vue:50]: We gotta fix this. I'll get in.

**Comment by boutell** [comment]
I pushed up two commits with the correct module relative paths, which look like:

`'Modules/@apostrophecms/modals/mixins/AposModalParentMixin'`

src/apos is implied, because what we're accessing here is the build folder containing the final frontend files for this module after all overrides are taken into account.

No other concerns!

**Pull request merged**